extensions/basic/jruby-ext/src/main/java/com/purbon/RubyBar.java

```diff
@@ -7,27 +7,48 @@
 import org.jruby.runtime.builtin.IRubyObject;
 
 /**
- * A basic class created for learning purpouses
+ * A basic class created for learning purposes
  * Created by purbon on 24/08/15.
  */
 
 @JRubyClass(name = "Bar")
-public class Bar extends RubyObject {
+public class RubyBar extends RubyObject {
 
-    public Bar(Ruby ruby, RubyClass metaclass) {
+    public RubyBar(Ruby ruby, RubyClass metaclass) {
         super(ruby, metaclass);
     }
 
+    /**
+     * Extension method with two names (aliases) that return a Hello World string). This is
+     * an static method.
```

Comment:
Equivalent to the following right? (I suggest adding into the docs here as well)

```ruby
class Bar
  class << self
    def shout()
      return "Hello World!"
    end
    alias_method :shout, :say
  end
end
```